database/tables/page_view_logs.pg

```diff
@@ -24,6 +25,7 @@ foreign-key constraints
     page_view_logs_assessment_id_fkey: FOREIGN KEY (assessment_id) REFERENCES assessments(id) ON UPDATE CASCADE ON DELETE CASCADE
     page_view_logs_assessment_instance_id_fkey: FOREIGN KEY (assessment_instance_id) REFERENCES assessment_instances(id) ON UPDATE CASCADE ON DELETE CASCADE
     page_view_logs_authn_user_id_fkey: FOREIGN KEY (authn_user_id) REFERENCES users(user_id) ON UPDATE CASCADE ON DELETE CASCADE
+    page_view_logs_client_fingerprint_id_fkey: FOREIGN KEY (client_fingerprint_id) REFERENCES client_fingerprints(id)
```

Comment:
We should be specifying ON UPDATE and ON DELETE behavior explicitly. And in the other tables.

Reply:
These have now been updated to specify `ON UPDATE` and `ON DELETE`.

Reply:
@mwest1066 is `CASCADE` the desired behavior here? I suppose it shouldn't really matter since we should never delete fingerprints.

Reply:
Good point: https://github.com/PrairieLearn/PrairieLearn/pull/8798#discussion_r1410072969